scripts/linter.py

```diff
@@ -19,7 +21,9 @@ def find_regex_in_file_content(
         with open(filename, 'r', encoding='utf-8') as file:
             for i, line in enumerate(file):
                 if search_pattern.search(line):
-                    print(f'{filename}:{i+1}: error: wrong notation "{search_pattern.search(line).string}"; expected "{correct_string}"')
+                    print(f"{filename}:{i+1}: error: wrong notation "
```

Comment:
Since there are going to be indented lines below, it's easier to read when they're all together as a block

```suggestion
                    print(
                        f"{filename}:{i+1}: error: wrong notation "
```

Reply:
done